SPECS/netdata.spec

```diff
@@ -394,167 +468,396 @@ if [ $1 -eq 0 ]; then
+%{_sbindir}/%{name}
+%{_sbindir}/%{name}-claim.sh
+%{_sbindir}/%{name}cli
+%if %{with log2journal}
+%{_sbindir}/log2journal
+%endif
+%{_sbindir}/systemd-cat-native
+%{_libexecdir}/%{name}/*
+%{_unitdir}/%{name}.service
+%{_tmpfilesdir}/%{name}.conf
+%caps(cap_dac_read_search,cap_sys_ptrace=ep) %attr(0750,root,netdata) %{_libexecdir}/%{name}/plugins.d/apps.plugin
+%caps(cap_setuid=ep) %attr(4750,root,netdata) %{_libexecdir}/%{name}/plugins.d/cgroup-network
+%attr(0750,root,netdata) %{_libexecdir}/%{name}/plugins.d/cgroup-network-helper.sh
+%caps(cap_setuid=ep) %attr(4750,root,netdata) %{_libexecdir}/%{name}/plugins.d/perf.plugin
+%caps(cap_setuid=ep) %attr(4750,root,netdata) %{_libexecdir}/%{name}/plugins.d/slabinfo.plugin
+%caps(cap_setuid=ep) %attr(4750,root,netdata) %{_libexecdir}/%{name}/plugins.d/xenstat.plugin
+%if %{with cups}
+%attr(0750,root,netdata) %{_libexecdir}/%{name}/plugins.d/cups.plugin
+%endif
+%exclude %{_libexecdir}/%{name}/edit-config
+%exclude %{_libexecdir}/%{name}/plugins.d/freeipmi.plugin
+%attr(0755, netdata, netdata) %{_localstatedir}/lib/%{name}
+%attr(0755, netdata, netdata) %dir %{_localstatedir}/cache/%{name}
+%attr(0755, netdata, netdata) %dir %{_localstatedir}/log/%{name}
 %config(noreplace) %{_sysconfdir}/logrotate.d/%{name}
+%attr(0750,root,netdata)%{_sbindir}/netdata-install-go-plugins.sh
+%config(noreplace) /etc/sysconfig/iptables_netdata
+%config(noreplace) /etc/sysconfig/ip6tables_netdata
 
-# systemd service or initscript
-%if %{with systemd}
-%{_unitdir}/netdata.service
-%else
-%{_sysconfdir}/rc.d/init.d/netdata
-%endif
+%files conf
+%doc README.md
+%license LICENSE REDISTRIBUTED.md
+%dir %{_sysconfdir}/%{name}
+%dir %{_sysconfdir}/%{name}/charts.d
+%dir %{_sysconfdir}/%{name}/health.d
+%dir %{_sysconfdir}/%{name}/python.d
+%dir %{_sysconfdir}/%{name}/statsd.d
+%dir %{_sysconfdir}/%{name}/go.d
+%config(noreplace) %{_sysconfdir}/%{name}/%{name}.conf.headless
+%dir %{netdata_conf_stock}/conf.d
+%{netdata_conf_stock}/conf.d/*
+%config(noreplace) %{_sysconfdir}/logrotate.d/netdata
+%config(noreplace) %{_sysconfdir}/profile.d/netdata.sh
+%dir %{_libexecdir}/%{name}
+%{_libexecdir}/%{name}/edit-config
+%{_sysconfdir}/netdata/.install-type
+
+%files data
+%doc README.md
+%license LICENSE REDISTRIBUTED.md
+%dir %{_datadir}/%{name}
+%attr(-, root, netdata) %{_datadir}/%{name}/web
+%{_datadir}/%{name}/go.d.checksums
 
-%{_libdir}/%{name}
-%{_sbindir}/%{name}
-%{_datadir}/%{name}
+%files freeipmi
+%doc README.md
+%license LICENSE REDISTRIBUTED.md
+%caps(cap_setuid=ep) %attr(4750,root,netdata) %{_libexecdir}/%{name}/plugins.d/freeipmi.plugin
 
-%attr(0770,netdata,netdata) %dir %{_localstatedir}/cache/%{name}
-%attr(0755,netdata,root) %dir %{_localstatedir}/log/%{name}
-%attr(0770,netdata,netdata) %dir %{_localstatedir}/lib/%{name}
-%attr(0770,netdata,netdata) %dir %{_localstatedir}/lib/%{name}/registry
+%changelog
+* Fri May 24 2024 Thierry Escande <thierry.escande@vates.tech> - 1.44.3-2
```

Comment:
Shouldn't we use the version `1.44.3-1.1` and not bumped their version so we won't have a conflict if we want to integrate their futur version `1.44.3-2`?

Reply:
Fixed